CODEOWNERS

```diff
@@ -26,6 +26,7 @@ src/audio/tone.c			@singalsu
 src/audio/kpb.c				@mrajwa
 src/audio/mux/*				@akloniex
 src/audio/dcblock*			@cujomalainey @dgreid
+src/audio/crossover*			@cujomalainey @dgreid
```

Comment:
Should you add yourself also?

Reply:
I am ending my internship next month and will not be working on the project anymore :) So for future references the CODEOWNERS should be @cujomalainey and @dgreid.

Reply:
If he comes back we can add him to the CODEOWNERs :)